### DISCUSSION TITLE
Purpose of the /storage/framework/cache/data directory

### DISCUSSION THREAD

**Discussion by nizarus** [firefly-iii/firefly-iii#9557] [Firefly III] [answered]
Hi,
What's the purpose of the /storage/framework/cache/data directory?
If it's just cache, can we remove its content ?
Thanks.

**Comment by JC5** [comment, marked as answer]
It's a cache directory. You can empty it if you want to :)

**Comment by nizarus** [comment]
Is it the content that is deleted by /flush?

**Comment by JC5** [comment]
What problem are you running into, please?

**Comment by nizarus** [comment]
My firefly instance is installed in a shared hosting with limited inodes :/ That's why I'm looking for any no needed files to remove them :)
When I try to remove cache with /flush I have a "Request Timeout" error.

**Reply by JC5** [reply]
Ahh yes. You can freely remove everything. The /flush routine does some general maintenance but much special. 

Can you use memcached or redis on your shared host?

**Reply by nizarus** [reply]
>  Can you use memcached or redis on your shared host?

No, the only cache module available is opcache
